.gitignore

```diff
@@ -0,0 +1,20 @@
+.DS_Store
```

Comment:
Yeah, I had forgot to add those. They are up there now if it means anything. My bad.

> On May 12, 2015, at 10:44 AM, Sean Golob <notifications@github.com> wrote:
> 
> In .gitignore <https://github.com/AaronMartone/http_server_with_simple_persistence/pull/1#discussion_r30160020>:
> 
> > @@ -0,0 +1,20 @@
> > +.DS_Store
> Tests?!?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/AaronMartone/http_server_with_simple_persistence/pull/1/files#r30160020>.
>